flink-cdc-common/src/main/java/org/apache/flink/cdc/common/utils/SchemaMergingUtils.java

```diff
@@ -85,6 +93,37 @@
  */
 @PublicEvolving
 public class SchemaMergingUtils {
+
```

Comment:
We may handle these changes in another PR and focus on supporting `DATE` and `TIME` formats here